x/bridge/types/keys.go

```diff
@@ -35,10 +39,21 @@ var (
 	BTCMintedKey = []byte{0x50}
 
 	BTCBurntKey = []byte{0x60}
+
+	// AssetsUnlockedSequenceTipKey is a standalone key for the assets unlocked sequence tip.
+	AssetsUnlockedSequenceTipKey = []byte{0x70}
+
+	// AssetsUnlockedSequenceKeyPrefix is the key prefix for the assets unlocked sequence.
+	AssetsUnlockedSequenceKeyPrefix = []byte{0x80}
```

Comment:
I would drop the "sequence" part. This is actually just the prefix of the key. Full key = prefix + sequence. 
```suggestion
	// AssetsUnlockedKeyPrefix is the prefix for the assets unlocked key.
	AssetsUnlockedKeyPrefix = []byte{0x80}
```
That would match `GetAssetsUnlockedKey` which then builds the key using `AssetsUnlockedKeyPrefix` and the given `sequence`.

Reply:
8e4ba3d0